engine/plugins/fbs/fbs.js

```diff
@@ -285,90 +286,76 @@ module.exports = function(options, imports, register) {
     // Defines the configuration for the online checker below.
     const onlineState = {
         online: true,
-        threshold: 5,
-        successfulOnlineChecks: 5,
-        onlineTimeout: 5000,
-        offlineTimeout: 30000,
-        ensureOnlineCheckTimeout: 300000
+        threshold: options?.config?.threshold ?? 0.5,
+        requestTimeout: options?.config?.requestTimeout ?? 2000,
+        circuitDuration: options?.config?.circuitDuration ?? 30000,
+        waitThreshold: options?.config?.waitThreshold ?? 10,
+        intervalDuration: options?.config?.intervalDuration ?? 5000
```

Comment:
The code that updates the configuration, seams to be missing: fbs.config

This config can change at run time and then needs to be updated...

Reply:
ups

Reply:
I these options change at runtime, circuit break will need to be setup again. Atm the options can not be changed from the administrationen. The options are set in config.json. Maybe we should just set a @todo about a future feature here?